api/sip/client.py

```diff
@@ -705,14 +707,22 @@ def read_message(self, max_size=1024*1024):
         data = ""
         tmp = ""
         while not done:
-            tmp = self.socket.recv(4096)
+            try:
+                tmp = self.socket.recv(4096)
+            except socket.timeout:
+                raise socket.error
+            except socket.error, e:
+                raise IOError("Socket error occurred receiving SIP response: %s" % e)
+            except Exception, e:
+                raise IOError("Some other error occurred with SIP response: %s" % e.message)
             data = data + tmp
-            if not tmp:
+            if not data:
```

Comment:
I agree that it doesn't make a difference but I think checking `tmp` was the right move.

Reply:
With the benefit of experienced eyes, then, leaving it a tmp is good.